scripts/interactions.js

```diff
@@ -308,7 +310,19 @@ damasflow_ondrop = function ( e )
 			}
 			else
 			{
-				if( confirm('Add ' + path + '?'))
+				if(oldPath===path){
+					var newWd=path.replace(/\/[^\/]*$/,"");
+					newWd=prompt("Ajouter � vos workdirs?",newWd);
```

Comment:
you assign different values and types to the same variable - this is error prone, can't be optimized by the interpreter and can be done on untyped language only. here you don't need an extra variable:
```js
prompt("Ajouter à vos workdirs?",path.replace(/\/[^\/]*$/,""));
```
try to avoid french.. until we have internationalization support we prefer to use english